.github/workflows/snyk-scan.yml

```diff
@@ -106,14 +105,14 @@ jobs:
         run: |
           sudo apt-get install -y libkrb5-dev
           pip install -r h2o-py/requirements.txt                  
-          snyk monitor -d --fail-on=all --file=h2o-py/requirements.txt --package-manager=pip --command=python3 --skip-unresolved --remote-repo-url=H2O-3 --project-name=H2O-3/${{ steps.extract_ref.outputs.ref }}/h2o-py/requirements.txt      
+          snyk monitor -d --fail-on=all --org=h2o-3 --file=h2o-py/requirements.txt --package-manager=pip --command=python3 --skip-unresolved --remote-repo-url=h2o-3/${{ steps.extract_ref.outputs.ref }} --project-name=H2O-3/h2o-3/${{ steps.extract_ref.outputs.ref }}/h2o-py/requirements.txt      
```

Comment:
Org names should be in cpaital right? H2O-3

Reply:
In the command the organisation name is case sensitive. It should be simple. But in Snyk dashboard organisation will be showed in capital letters. So there is no problem.

Following is an example I encountered in nitro repository.
![image](https://user-images.githubusercontent.com/67860730/218742795-70c44160-989e-4368-b8db-694ce917b522.png)

Reply:
got it